payment-service/src/main/java/com/crmfoodestablishment/paymentservice/entity/PaymentStage.java

```diff
@@ -0,0 +1,17 @@
+package com.crmfoodestablishment.paymentservice.entity;
+
+public enum PaymentStage {
+    UNPAID("Unpaid"),
```

Comment:
are u sue that we need constructor here? and getDisplayNamemethod?

Reply:
done